src/darwin/Framework/CHIPTests/TestHelpers/MTRTestServerAppRunner.h

```diff
@@ -48,8 +48,10 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  * Same thing, but initialize as a "cross test" helper, which is not killed at
  * the end of the current test (but is killed at the end of the current suite).
+ *
+ * The passed-in testsuite must be a subclass of MTRTestCase.
  */
-- (instancetype)initCrossTestWithAppName:(NSString *)name arguments:(NSArray<NSString *> *)arguments payload:(NSString *)payload testcase:(MTRTestCase *)testcase;
+- (instancetype)initCrossTestWithAppName:(NSString *)name arguments:(NSArray<NSString *> *)arguments payload:(NSString *)payload testsuite:(Class)testsuite;
```

Comment:
Instead of passing around the test case class and having different initializers for suite vs test scope, have you considered making this functionality available as a category on MTRTestCase?

Reply:
Hmm, without involving an apprunner object at all?  Let me think about it.

Reply:
Yeah, this is a good idea; I'll do that.